src/pages/docs/class-naming-convention.md

```diff
@@ -16,22 +16,27 @@ One of the most annoying things that happens when you use a CSS library is class
 
 To avoid this, the Spark Design System uses a global namespace to ensure that it's styles don't interfere with the custom styles in your app.
 
-## Class prefixes
+## Class Prefixes
 After the global namespace, each class name has a prefix which gives information about what the class is doing. The prefixes available are:
 
 * **`b-`** (Base) For classes that add additional style to base HTML elements.
 * **`o-`** (Object) Cosmetic-free design patterns that are very dangerous to change because they are often used in unrelated contexts. E.g. the OOCSS Media Object.
 * **`c-`** (Component) Implementation-specific pieces of UI. CSS is safe to change because it is isolated to the specific component.
 * **`u-`** (Utility) Highly specific, highly reusable, usually single purpose, and have high specificity.
 * **`is-`, `has-`** (State) These classes are typically added and removed through JavaScript or on the server to show specific states.
-* **`js-`** (JavaScript) For targeting JavaScript specific functionality. No styles should be bound to these classes. They are only meant for behavior.
 
 ## Pascal Casing
 After the namespace and prefix, the main part of the class name doesn't stand out very well. For this reason we use pascal case to visually separate it from the rest of the name. For example:
 ```
 .sprk-c-HighlightBoard__content
 ```
 
+## Class Suffixes
+Responsive suffixes on classes are utilized when the need to have styles start at a specific breakpoint occurs. The `@` symbol must be escaped in the stylesheet. For example:
```

Comment:
This sentence is funky. Maybe: "Responsive suffix classes are used when you need styles to be applied at specific breakpoints."